modules/nw-sriov-networknodepolicy-object.adoc

```diff
@@ -46,6 +46,13 @@ spec:
 When specifying a name, be sure to use the accepted syntax expression `^[a-zA-Z0-9_]+$` in the `resourceName`.
 
 <4> The node selector specifies the nodes to configure. Only SR-IOV network devices on the selected nodes are configured. The SR-IOV Container Network Interface (CNI) plugin and device plugin are deployed on selected nodes only.
++
+[IMPORTANT]
+====
+If a node is in an unhealthy machine config pool, the process of applying node network configuration policies to all targeted nodes in the cluster will pause until the machine config pool returns to a healthy state. This interruption occurs because the SR-IOV Network Operator checks that a node's MCP is not in a `Degraded` or `Updating` state before applying a policy and the SR-IOV Network Operator applies policies to nodes in sequence.
+
+To avoid a node in an unhealthy machine config pool from blocking the application of node network configuration policies to other nodes, including nodes in other machine config pools, you must create a separate node network configuration policy for each machine config pool. This segregation ensures that an unhealthy state for one machine config pool does not prevent the network configuration of nodes in other pools.
```

Comment:
Great review feedback @xenolinux , thanks. The only bit I didn't implement fully was the second paragraph, as I wanted to highlight that it would block nodes in other MCPs too. But I shortened it to:

To avoid a node in an unhealthy MCP from blocking the application of node network configuration policies to other nodes, including nodes in other MCPs, you must create a separate node network configuration policy for each MCP.

Reply:
No worries. Sounds good :)